app/services/create_requested_account.rb

```diff
@@ -26,6 +26,19 @@ def initialize(requested_account, creator)
     process_request(@creator, @creation_reason)
   end
 
+  def set_result_description
+    if @result[:failure]
+      @result[:result_description] = @result[:failure].sub('response: {}', '')
+                                                      .sub('https://en.wikipedia.org', '')
+      @result[:result_description] = I18n.t('users.requested_account_status.failure_message',
+                                            result_description: @result[:result_description])
+    elsif @result[:success]
+      @result[:result_description] = @result[:success]
```

Comment:
Same as above, writing twice to `@result[:result_description]` is a little confusing. Here, you can just skip this steppass `@result[:success]` to `I18n.t` .

Reply:
Done!